core/src/main/java/de/javakaffee/web/msm/StorageClientFactory.java

```diff
@@ -43,6 +43,15 @@ MemcachedClient createCouchbaseClient(MemcachedNodesManager memcachedNodesManage
                 long maxReconnectDelay, Statistics statistics );
     }
 
+    static interface MemcachedNodeURLBasedConnectionFactory {
```

Comment:
Rename to `MemcachedConnectionFactory`? The implementation class `MemcachedConnectionFactory` could then be renamed to e.g. `StandardMemcachedConnectionFactory`.